test/integration/PaymentTypes/PaylaterInvoiceTest.php

```diff
@@ -186,8 +184,8 @@ public function paylaterInvoiceCanbeChargedWithShippingData()
 
         $fetchedCharge = $this->unzer->fetchChargeById($authorization->getPaymentId(), $chargeResponse->getId());
         $this->assertEquals(
```

Comment:
This should also be compared to the original shippingData.
What if the charge response has been empty and the fetched too, even though shippingData has been added in the first place?